library/cwm/examples/object_api.rb

```diff
@@ -0,0 +1,70 @@
+# Simple example to demonstrate object API for CWM
+
+require_relative "example_helper"
+
+require "cwm/widget"
+
+Yast.import "CWM"
+Yast.import "Wizard"
+
+class LuckyNumberWidget < CWM::IntField
+  attr_reader :result, :minimum, :maximum
+
+  def initialize
+    @minimum = 0
+    @maximum = 1000
+    self.widget_id = "lucky_number"
```

Comment:
hmm, yes good idea, in general you usually do not need widget_id as long as it is unique